src/main/java/com/amazon/dlic/auth/http/jwt/HTTPJwtAuthenticator.java

```diff
@@ -67,7 +68,8 @@ public class HTTPJwtAuthenticator implements HTTPAuthenticator {
     public HTTPJwtAuthenticator(final Settings settings, final Path configPath) {
         super();
 
-        String signingKey = settings.get("signing_key");
+        List<String> signingKeys = settings.getAsList("signing_key");
```

Comment:
Thanks for the change @stephen-crawford ! Can we add an issue to update docs how users can rotate their keys?

Also - an example would help on how multiple keys can be specified. Looks like getAsList supports multiple different ways- https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/common/settings/Settings.java#L426-L448

Reply:
+1, can we use the integrationTest framework to add a test for this there as well?

Reply:
Sure.